apps/service_providers/tracing/service.py

```diff
@@ -265,3 +265,10 @@ def _get_current_span_info(self) -> tuple[UUID, str]:
         if self.span_stack:
             return self.span_stack[-1]
         return self.trace_id, self.trace_name
+
+    def add_output_message_tags_to_trace(self, tags: list[str]) -> None:
```

Comment:
These should be added to the langfuse trace directly: https://python.reference.langfuse.com/langfuse/api#Trace.tags

So this needs to call each tracer in `self._active_tracers` and let them handle the tags